DOC-1960: The urlinput dialog component would not open the type-ahead dropdown when the input value was reset to an empty string. (#2683)
tinymce/tinymce-docs: modules/ROOT/pages/6.4.2-release-notes.adoc

// {productname} 6.4.2 also includes the following bug fixes:

> === The urlinput dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.

Reviewer: ```suggestion
=== The `urlinput` dialog component would not open the type-ahead dropdown when the input value was reset to an empty string.
```

In previous versions of {productname}, a console error would be thrown when resetting the `Source` field of the **Insert Media** dialog.

> As a consequence, the dropdown list with the url suggestion would not expand.

Reviewer: ```suggestion
As a consequence, the dropdown list with the URL suggestions would not expand.
```

To fix this issue, {productname}, updated the code for the `urlinput` component that is responsible for gathering the correct dropdown items.

> As a result, the dropdown list with URL suggestions now expands even on the empty Source field and the browser console is cleaned out of errors.

Reviewer: ```suggestion
As a result, the dropdown list with URL suggestions now expands even on the empty `Source` field, and the browser console is cleaned out of errors.
```

> In previous versions of {productname}, a console error would be thrown when resetting the `Source` field of the **Insert Media** dialog.
> 
> As a consequence, the dropdown list with the URL suggestions would not expand.
> 
> To fix this issue, {productname}, updated the code for the `urlinput` component that is responsible for gathering the correct dropdown items.
> 
> As a result, the dropdown list with URL suggestions now expands even on the empty `Source` field, and the browser console is cleaned out of errors.

Reviewer 2: ```suggestion
In previous versions of {productname}, a console error was thrown when the _Source_ field of the *Insert Media* dialog was reset.

As a consequence, the dropdown list of suggested URLs did not expand.

To fix this issue, the code for the `urlinput` component that is responsible for gathering the correct dropdown items was updated.

As a result, the URL suggestions dropdown list now expands even on the empty _Source_ field. As well, the host browser’s console no longer returns errors.
```

Copy-edits and some minor edits.
